### PR TITLE
Initial Inventory Examples

### PR DESCRIPTION
**Pull request by italobusi**
Initial examples for network inventory management: see #110

Based on the slides in [Examples for Scenarios from VDF - ib.pptx](https://github.com/ietf-ccamp-wg/transport-nbi/files/13744449/Examples.for.Scenarios.from.VDF.-.ib.pptx)

Note: while compiling the JSON examples some errors in the 'ietf-network-inventory' and 'ietf-ni-location' modules have been spotted. These have been reported either by new open issues in [network-inventory-yang](https://github.com/ietf-ivy-wg/network-inventory-yang) or by a mail to the [draft-wbbpb-ivy-network-inventory-location](https://datatracker.ietf.org/doc/html/draft-wbbpb-ivy-network-inventory-location) I-D authors.

### CODE REVIEW

**Comment by italobusi** [comment]
Note that some gaps have been identified in the current network inventory YANG model

Attributes missing in the current network inventory YANG model are provided in the example in https://github.com/ietf-ccamp-wg/transport-nbi/pull/114/commits/17327bcd8185acf7187d1acfeae16b4a55f75db5 and marked with the prefix 'tba'

**Review by italobusi — COMMENTED** [review]
Comments from the DT call on 2024-01-18:

1) Replace ne-id, component-id, room-id and rack-id with UUIDs to avoid misinterpretations

   A possible web site to be used for generating UUIDs is: https://www.uuidgenerator.net/version4

2) The location string is not formatted according to the pattern defined in ONF TR-547: example to be updated

File: json-examples/inventory-examples.json [hunk 1]
```diff
@@ -0,0 +1,365 @@
...
+              "rack-id": "e7769bf4-3334-4506-9ea6-da9104e82675",
+              "name": "rack-1-1",
+              "description": "Rack 1 in row 1 of room Wuhan-C-A1-NO.101. Fully occupied by Ne(115-50).",
+              "rack-location": {
+                "row-number": 1,
+                "column-number": 1
+              },
+              "contained-chassis": [
+                {
+                  "relative-position": 1,
```
Comment (italobusi) [inline json-examples/inventory-examples.json:179]:
```suggestion
                  "rack-relative-position": 1,
```

File: json-examples/inventory-examples.json [hunk 2]
```diff
@@ -0,0 +1,365 @@
...
+        "description": "NE with multiple shelves occupying a full rack",
+        "components": {
+          "component": [
+            {
+              "component-id": "5d3bbcba-ad7a-4e55-bf0d-003bb23562c8",
+              "name": "A1-101-1-1-2",
+              "description": "Chassis 2 of rack 1 in row 1 of room Wuhan-C-A1-NO.101",
+              "class": "ianahw:chassis",
+              "parent-rel-pos": 1,
+              "tba:location": "r=1, sh=1"
```
Comment (italobusi) [inline json-examples/inventory-examples.json:16]:
```suggestion
              "tba:location": "Ne(115-50), r=1, sh=1"
```

**Comment by italobusi** [comment]
I have just created an issue for the missing attribute: https://github.com/ietf-ivy-wg/network-inventory-yang/issues/42

**Review by YuChaode — APPROVED** [review]

**Pull request merged**